.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
```

Comment:
[suggest] Considering consistency with other workflows, `...ing` seems better:

```suggestion
name: Releasing
```

And `release.yml` → `releasing.yml`.

Reply:
Sure.